components/BoardDetailInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import styles from "./BoardDetail.module.css";
+import profile from "../public/profile.svg";
+import heart from "../public/heart.svg";
+import { Article } from "./type";
+
+interface Props {
+  article: Article | null;
+}
+
+export default function BoardDetailInfo({ article }: Props) {
+  if (!article) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <>
+      <section className={styles["detail-wrapper"]}>
+        <h1 className={styles.h1}>{article.title}</h1>
+        <div className={styles["detail-wrapper-info"]}>
+          <div className={styles["detail-wrapper-user"]}>
+            {article.image ? (
+              <Image src={article.image} width="24" height="24" alt="프로필" />
+            ) : (
+              <Image src={profile} width="24" height="24" alt="프로필" />
+            )}
```

Comment:
삼항 연산자를 컴포넌트 외부가 아닌 src 속성 안에서의 조건으로 활용하면 더 간단히 작성할 수 있습니다. 😁
```suggestion
             <Image src={article.image ? article.image : profile} width="24" height="24" alt="프로필" />
```

Reply:
또한, `||` 연산자를 활용하면 더 간단히 작성할 수도 있습니다.

```ts
<Image src={article.image || profile} width="24" height="24" alt="프로필" />. 
```